app/controllers/neighborhoods_controller.rb

```diff
@@ -46,6 +46,15 @@ def update
     end
   end
 
+  def destroy
+    @neighborhood = Neighborhood.find(params[:id])
+    if current_user.admin?
```

Comment:
consider an alternative:

```ruby
@neighborhood.destroyable_by?(current_user)
```

this makes it a model behavior that can be easily unit tested